src/components/Icon/Icon.tsx

```diff
@@ -75,7 +76,7 @@ export const Icon: React.FC<IconProps> = ({
       result.push(iconClass);
     }
 
-    return result.join(" ");
+    return result.join(" ").trimStart();
```

Comment:
just use `trim` no reason to only get one end
```suggestion
    return result.join(" ").trim();
```

Reply:
for bonus points, convert this using a `switch` to using `clsx` (which is what we should have done from the start 😬, but hindsight is 20-20)

Reply:
✅